README.md

```diff
@@ -51,9 +62,9 @@ This installer is incompatible with `default` namespace installations, install a
 >- [Pre-existing Developer Hub Instances](./docs/RHDH-CONFIG.md)
 
 For convenience you can configure your Red Hat Developer Hub, GitOps, and Pipelines Operators to enable the use of AI Software Templates by running the following from the root of this repository:
-
-`bash ./configure.sh`
-
+```
+bash ./configure.sh
+```
 **Note**: If you changed the installation namespace used by the installer you will first need to run `export NAMESPACE=<namespace used>` as the default value is `ai-rhdh`.
 
 See the following for further customization of the configuration:
```

Comment:
Can we end the main README over here? And cut paste everything below to a new doc `INSTALLER-CUSTOMIZATION.yaml`?

This line has redirects bullet points anyways, so we can further clean it up and be more readable :)

Reply:
We can end the README there, are we good with the spin-off file being named something like `CONFIG.md`? Since it would contain items such as 
- Lightspeed config
- GitLab
- Setting env vars
- Setting catalog

Reply:
yes i like it